modular_skyrat/modules/loadouts/loadout_items/under/loadout_datum_under.dm

```diff
@@ -390,6 +391,14 @@ GLOBAL_LIST_INIT(loadout_miscunders, generate_loadout_items(/datum/loadout_item/
 	name = "Villain Suit"
 	item_path = /obj/item/clothing/under/costume/villain
 
+/datum/loadout_item/under/miscellaneous/costume/vic_dress
+	name = "Black victorian dress"
+	item_path = /obj/item/clothing/under/costume/vic_dress
+
+/datum/loadout_item/under/miscellaneous/costume/vic_dress_red
```

Comment:
these should be under the formal stuff, tbh

Reply:
Probably not wrong, I just allocated it based on it's item path

Reply:
idk
its more a formal getup to me but ydy

Reply:
I'll definitely take a look and sanity check it's location before I mark as resolved

Reply:
resolved in 158281a